### PR TITLE
Fix GameTitles(JPN) & Set Speed Hack on 2titles

### PR DESCRIPTION
**Pull request by nassau-tk**
### Description of Changes

Fix Japanese Title (Trial,Demo Section)
Add 2titles Speed Hack

SLPS-20308:
name: "花火百景 [特典版]"
name-sort: "はなびひゃっけい [とくてんばん]"
name-en: "Hanabi Hyakkei [Tokutenban]"
region: "NTSC-J"
speedHacks:
instantVU1: 0 # Fixes working speed to correctly.
mtvu: 0 # Fixes working speed to correctly.

SLPS-20309:
name: "花火百景"
name-sort: "はなびひゃっけい"
name-en: "Hanabi Hyakkei"
region: "NTSC-J"
speedHacks:
instantVU1: 0 # Fixes working speed to correctly.
mtvu: 0 # Fixes working speed to correctly.

These title must to turn off the options.
Becasue, If turn on the option then the game can't work correct speed.
*These title is same game means [Limited & Standard Edition]

SLPS-25481:
name: "ガッツだ！！森の石松"
name-sort: "がっつだ もりのいしまつ"
name-en: "Gattsu Da！！ Mori no Ishimatsu"
region: "NTSC-J"
speedHacks:
instantVU1: 0 # Fixes Turbo mode Freeze.
mtvu: 0 # Fixes Turbo mode Freeze.

If turn on the options then the game will freeze by Turbo mode.
If turn off the options then Turbo mode wil work stable.
### Rationale behind Changes

Japanese titles should appear by Japanese I think.

### Suggested Testing Steps

See above

### CODE REVIEW

**Comment by nassau-tk** [comment]
SLPM-74105:
  name: "桃太郎電鉄16 北海道大移動の巻 [PlayStation2 the Best]"
  name-sort: "ももたろうでんてつ16 ほっかいどうだいいどうのまき [PlayStation2 the Best]"
  name: "Momotarou Dentetsu 16 - Hokkaido Daiidou no Maki！ [PlayStation2 the Best]"
  region: "NTSC-J"

SLPM-62750:
  name: "桃太郎電鉄16 北海道大移動！の巻"
  name-sort: "ももたろうでんてつ16 ほっかいどうだいいどう！のまき"
  name: "Momotarou Dentetsu 16 - Hokkaido Daiidou no Maki！ [PlayStation2 the Best]"
  region: "NTSC-J"

I miss it!!

name-en:

**Comment by nassau-tk** [comment]
I had many miss take.
But, I fix it.
Please, Check and merge it.
Thank you.

File: bin/resources/GameIndex.yaml [hunk 1]
```diff
@@ -9915,7 +9915,7 @@ SCPS-55001:
 SCPS-55002:
   name: "零 ～zero～"
   name-sort: "ぜろ"
-  name-en: "Fatal Frame"
+  name-en: "Zero"
```
Comment (Mrlinkwii) [inline bin/resources/GameIndex.yaml:9918]: can you change this back for me

File: bin/resources/GameIndex.yaml [hunk 2]
```diff
@@ -60477,20 +60792,20 @@ SLPS-73254:
 SLPS-73255:
   name: "零 ～zero～ [PlayStation2 the Best]"
   name-sort: "ぜろ [PlayStation2 the Best]"
-  name-en: "Fatal Frame [PlayStation 2 the Best - Reprint]"
+  name-en: "Zero [PlayStation 2 the Best - Reprint]"
```
Comment (Mrlinkwii) [inline bin/resources/GameIndex.yaml:60795]: again dont change this

File: bin/resources/GameIndex.yaml [hunk 3]
```diff
@@ -24572,8 +24564,6 @@ SLES-54152:
   name: "The Ant Bully"
   name-sort: "Ant Bully, The"
   region: "PAL-M5"
-  gsHWFixes:
-    nativeScaling: 2 # Fixes misaligned post effects.
```
Comment (Mrlinkwii) [inline bin/resources/GameIndex.yaml:24576]: why was this removed ?

**Comment by JordanTheToaster** [comment]
The entire PR is pretty messed up and touches many things it shouldn't be.

**Comment by nassau-tk** [comment]
It will removed by time lag I think.
Cuz, l never delete it.

oK.

I'm very tired by this work. lol

I can't check these diff and there needs now.

I close this PR.

I will re-push only speed hack later.

Thak you every one!!
